src/main/java/edu/greenblitz/robotName/subsystems/Lifter/LifterConstants.java

```diff
@@ -0,0 +1,22 @@
+package edu.greenblitz.robotName.subsystems.Lifter;
+
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+
+public class LifterConstants {
+    public static final int MOTOR_ID = 1;
+    public static final double PID_KP = 3.6;
+    public static final double PID_KI = 1;
+    public static final double PID_KD = 0;
```

Comment:
instead of wiritng every one put it just in PIDController

Reply:
we were told not to do so

Reply:
by who

Reply:
@tnrz135: +1 to the "by  who"